rllib/evaluation/sampler.py

```diff
@@ -758,6 +838,44 @@ def _process_observations(
                     infos=infos[env_id].get(agent_id, {}),
                     new_obs=filtered_obs,
                     **episode.last_pi_info_for(agent_id))
+                perf_stats.storing_samples_in_buffer += time.time() - t
+            elif _use_trajectory_view_api:
```

Comment:
Is it possible to, instead of adding if/elses, have separate _process_observation etc. functions for traj view mode?

Reply:
sure ...

Reply:
done